src/System.Azure.Experimental/System/Azure/StorageAccessSignature.cs

```diff
@@ -6,29 +6,31 @@
 using System.Text.Utf8;
 using System.Binary.Base64Experimental;
 using System.Buffers.Text;
+using System.Buffers;
 
 namespace System.Azure.Authentication
 {
     public static class StorageAccessSignature
     {
         static Utf8String s_emptyHeaders = (Utf8String)"\n\n\n\n\n\n\n\n\n\n\nx-ms-date:"; // this wont be needed once we have UTF8 literals
+        static TransformationFormat s_toBase64 = new TransformationFormat(Base64Experimental.BytesToUtf8Encoder);
```

Comment:
nit: add explicit `private` access modifier, here and elsewhere

Reply:
will do